freqtrade/arguments.py

```diff
@@ -172,7 +174,7 @@ def optimizer_shared_options(parser: argparse.ArgumentParser) -> None:
     @staticmethod
     def hyperopt_options(parser: argparse.ArgumentParser) -> None:
         """
-        Parses given arguments for Hyperopt scripts.
+        Parses given arguments foér Hyperopt scripts.
```

Comment:
shouldn't it be `for` 😬

Reply:
What I did again :disappointed_relieved:
Will be fixed